libs/designer-ui/src/lib/chatbot/components/chatBubble.tsx

```diff
@@ -71,6 +73,16 @@ export const ChatBubble: React.FC<ChatBubbleProps> = ({
             <div className={'msla-bubble-footer-disclaimer'}>{intlText.aIGeneratedDisclaimer}</div>
             {onThumbsReactionClicked && (
               <div className={'msla-bubble-reactions'}>
+                {text && (
```

Comment:
we should also move this above the onThumbsReactionClicked component on line 74 that way it doesn't get hidden if we don't pass in the callback to this component

Reply:
done!